tikv-client/src/main/java/com/pingcap/tikv/meta/TiTableInfo.java

```diff
@@ -44,7 +44,7 @@
   private final long maxColumnId;
   private final long maxIndexId;
   private final long oldSchemaId;
-  private final long columnLength;
+  private final long columnSize;
```

Comment:
pls add comments, what `columnSize` means?

Reply:
estimated column size.

Reply:
pls add the comment in source code

Reply:
done.